packages/common/src/sites/_internal/computeLinks.ts

```diff
@@ -0,0 +1,37 @@
+import { IItem } from "@esri/arcgis-rest-types";
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { UserSession } from "@esri/arcgis-rest-auth";
+import { IHubEntityLinks } from "../../core/types";
+import { getItemIdentifier } from "../../items";
+import { getRelativeWorkspaceUrl } from "../../core/getRelativeWorkspaceUrl";
+import { getItemThumbnailUrl } from "../../resources/get-item-thumbnail-url";
+import { getHubRelativeUrl } from "../../content/_internal/internalContentUtils";
+
+/**
+ * Compute the links that get appended to a Hub Site
+ * search result and entity
+ *
+ * @param item
+ * @param requestOptions
+ */
+export function computeLinks(
```

Comment:
Took the opportunity to abstract this into a util like we're doing for other entities - this ensures that the links that are computed when we enrich site search results are the same as the ones we append to the entity in `computeProps`